state-chain/pallets/cf-ingress-egress/src/tests/boost.rs

```diff
@@ -836,7 +825,7 @@ fn boosting_deposits_is_disabled_by_safe_mode() {
 			BoostPoolTier::FiveBps
 		));
 
-		turn_safe_mode_on();
+		toggle_boosting_with_safe_mode(false);
```

Comment:
nit, toggle suggests if it's false, then make it true and vice vesrsa, rather than providing a value to set it
```suggestion
		boosting_with_safe_mode(false);
```